src/admin-views/aggregator/origins/refine.php

```diff
@@ -57,24 +44,25 @@
  *
  * @param array $keyword_exclusions List of origins excluded.
  */
-$keyword_exclusions = json_encode( apply_filters( 'tribe_events_aggregator_refine_keyword_exclusions', $keyword_exclusions ) );
+$keyword_exclusions = apply_filters( 'tribe_events_aggregator_refine_keyword_exclusions', [ 'facebook' ] );
+$keyword_exclusions = json_encode( $keyword_exclusions );
```

Comment:
:warning: json_encode() is discouraged. Use wp_json_encode() instead.

<sub>hash: acf70da3daa51e852c0f36dc01930918</sub>